### ISSUE TITLE
KeyError when force calling

### ISSUE THREAD

**Issue by wdecoster** [tjiangHIT/cuteSV#56]
Hi,

I did the following with cuteSV 1.0.10:

- CuteSV SV calling
- merging the VCFs with Jasmine
- CuteSV force calling with -Ivcf

However, the last step generates the following error:

```
Traceback (most recent call last):
  File "/home/wdecoster/p200/workflow_results/locustype/.snakemake/conda/ab0f7b26/bin/cuteSV", line 788, in <module>
    run(sys.argv[1:])
  File "/home/wdecoster/p200/workflow_results/locustype/.snakemake/conda/ab0f7b26/bin/cuteSV", line 784, in run
    main_ctrl(args, argv)
  File "/home/wdecoster/p200/workflow_results/locustype/.snakemake/conda/ab0f7b26/bin/cuteSV", line 658, in main_ctrl
    result = force_calling(args.input, args.Ivcf, args.output, temporary_dir,
  File "/home/wdecoster/p200/workflow_results/locustype/.snakemake/conda/ab0f7b26/lib/python3.8/site-packages/cuteSV/cuteSV_forcecalling.py", line 350, i
    for strand_iter in sv_dict['INV'][chrom]:
KeyError: 'chr15'
```

Regards,
Wouter

**Comment by Meltpinkg** [comment]
Hello, @wdecoster 

Thanks for using cuteSV.
The bug you mentioned above has been fixed in cuteSV 1.0.11. Please update cuteSV and have a try again.

Best,
Shuqi

**Comment by wdecoster** [comment]
Hi Shuqi,

Thanks for the quick reply, updating did solve the issue!

Cheers,
Wouter
